modules/RouterUtils.js

```diff
@@ -1,26 +1,7 @@
-import deprecateObjectProperties from './deprecateObjectProperties'
-
 export function createRouterObject(history, transitionManager) {
   return {
     ...history,
     setRouteLeaveHook: transitionManager.listenBeforeLeavingRoute,
```

Comment:
I think I'd pull a fast one on users and switch it to the `tm.listen`, it's the one you want in the render tree, not history's, and if you want history's, you already have it elsewhere. We don't have to provide BC for *everything*. Stuff like this I'd rather just cut over, I can't imagine an app would need to change their code in very many places.